tensorboard/data/experimental/BUILD

```diff
@@ -0,0 +1,73 @@
+# Description:
+# Experiment Data Access API.
+
+package(default_visibility = ["//tensorboard:internal"])
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+py_library(
+    name = "experimental",
+    srcs = ["__init__.py"],
```

Comment:
We generally do this only for public APIs, and this is explicitly
intended to *not* yet be a public API; should we remove this and the
`__init__.py`?